src/jsonapi.js

```diff
@@ -95,27 +95,29 @@ export const createEntity = (entity, {
   return (dispatch, getState) => {
     dispatch(apiWillCreate(entity));
 
-    const { host: apiHost, path: apiPath, accessToken } = getState().api.endpoint;
-    const endpoint = `${apiHost}${apiPath}/${entity.type}`;
+    const { axiosConfig } = getState().api.endpoint;
+    const endpoint = `${axiosConfig.baseURL}/${entity.type}`;
+
+    const options = {
+      ...axiosConfig,
+      method: 'POST',
+      body: JSON.stringify({ data: entity })
```

Comment:
As far as I can see from the Axios docs, this attributes should be `data` and doesn't need to be stringified: https://github.com/mzabriskie/axios#axiosconfig